### PR TITLE
Add option to specify cluster version

### PR DESCRIPTION
**Pull request by hhorak**
In order to allow parametrize tests, we need to allow to choose version of the cluster (server) to be used, no matter what version the `oc` tool is. That is possible by setting a variable `OC_CLUSTER_VERSION` (parameter of the function `ct_os_cluster_up` may be also used), which results in parameter `--version=vX.Y` of the `oc cluster up` call.

### CODE REVIEW

File: test-lib-openshift.sh [hunk 1]
```diff
@@ -294,11 +294,16 @@ function ct_os_install_in_centos() {
 # Arguments: is_public - sets either private or public hostname for web-UI,
 #                        use "true" for allow remote access to the web-UI,
 #                        "false" is default
+# Arguments: cluster_version - version of the OpenShift cluster to use, empty
```
Comment (praiskup) [inline test-lib-openshift.sh]: The new argument is missing in `ct_os_cluster_up [DIR, IS_PUBLIC]` header.

**Comment by praiskup** [comment]
small nit, otherwise lgtm, feel free to push directly

**Comment by praiskup** [comment]
Well, it is preferred to have the comment also in the git commit message (not only on github page).

**Comment by hhorak** [comment]
Thanks for the feedback, fixed as suggested.

**Pull request merged**